examples/client-logger/app-insights-logger/jest.config.js

```diff
@@ -13,4 +13,8 @@ module.exports = {
 	testPathIgnorePatterns: ["/node_modules/", "dist"],
 	moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
 	coveragePathIgnorePatterns: ["/node_modules/", "/src/test/"],
+	testEnvironment: "jsdom",
```

Comment:
Newer versions of jest default to the test node environment.

This was done to split jsdom support into a separate package, reducing jest's package size.